src/manifest/dbtProject.ts

```diff
@@ -651,11 +651,11 @@ export class DBTProject implements Disposable {
   }
 
   showCompiledSql(modelPath: Uri) {
-    this.findModelInTargetfolder(modelPath, "compiled");
+    this.findModelInTargetfolder(modelPath);
   }
 
   showRunSQL(modelPath: Uri) {
-    this.findModelInTargetfolder(modelPath, "run");
+    this.findModelInTargetfolder(modelPath);
   }
```

Comment:
the `type` cannot be removed as it is used to find files in compiled or run directory